### PR TITLE
f/435 adapt treemap to new encoding data

### LINKED ISSUE #435: Show compression type in treemap
**Please describe the solution you'd like:**  
Show the used compression type in the treemap to improve insights - e.g. Dictionary (FSBA).

**Affected Component(s):**  
`DatabaseManager`, `Cockpit`, `Frontend`

as reported by @Bensk1

### LINKED ISSUE #436: Visualize encoding types in percentage
**Please describe the solution you'd like:**  
Visualize the used encodings e.g. 60%: Dictionary (FSBA), 40%: LZ4 (SIMDBP128)

**Affected Component(s):**  
`DatabaseManager`, `Cockpit`, `Frontend`

**Additional context:**  
Feel free to add some more description as I can't quite grab it yet, @Bensk1

### PR DESCRIPTION
**Pull request by MaxSchneider1337**
# Resolves #435 and resolves #436 

**Does your pull request add a feature? Please describe:**  
shows the encoding in percentage and all the compression information. 

**Affected Component(s):**  
TransformationService

### CODE REVIEW

File: hyrisecockpit/frontend/src/services/transformationService.ts [hunk 1]
```diff
@@ -135,6 +135,46 @@ function getStorageData(data: any, primaryKey: string = ""): StorageData {
...
+            compressionArray: [number]
+          ) =>
+            text +
+            compression +
+            (compressionIndex === compressionArray.length ? "" : ", "),
+          ""
+        );
+        return (
+          encodingText +
+          currentEncoding.amount / totalAmount +
```
Comment (schTi) [inline hyrisecockpit/frontend/src/services/transformationService.ts]: Can you ensure two decimal digits here? Just use `roundNumber` function for that

File: hyrisecockpit/frontend/src/services/transformationService.ts [hunk 2]
```diff
@@ -135,6 +135,46 @@ function getStorageData(data: any, primaryKey: string = ""): StorageData {
...
+    );
+
+    return rawData.reduce(
+      (
+        encodingText: string,
+        currentEncoding: any,
+        encodingIndex: number,
+        encodingArray: [any]
+      ) => {
+        const compressionText = currentEncoding.compression.reduce(
```
Comment (schTi) [inline hyrisecockpit/frontend/src/services/transformationService.ts:151]: Multiple reduce on string looks crazy :smile:

**Review by schTi — CHANGES_REQUESTED** [review]
Good stuff

File: hyrisecockpit/frontend/src/services/transformationService.ts (hunk 1 above)
Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/services/transformationService.ts]: used `getPercentage` for that.

**Comment by MaxSchneider1337** [comment]
thank you @schTi

**Review by schTi — APPROVED** [review]
:fire:

**Pull request merged**